.github/workflows/test.yaml

```diff
@@ -214,6 +288,39 @@ jobs:
         name: coverage
         path: coverage.postgres.out
 
+  integration-tests-postgres-nightly:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: setup golang
+      uses: actions/setup-go@v3
+      with:
+        go-version: '^1.18'
+
+    - name: cache go modules
+      uses: actions/cache@v3
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-build-codegen-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-build-codegen-
+
+    - name: checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: run integration tests
+      run: make test.integration.postgres
+      env:
+        TEST_KONG_IMAGE: "kong/kong"
+        TEST_KONG_TAG: "latest"
+
+    - name: collect test coverage
+      uses: actions/upload-artifact@v3
+      with:
+        name: coverage
+        path: coverage.postgres.out
   integration-tests-feature-gates:
     runs-on: ubuntu-latest
     steps:
```

Comment:
Running our nightlies against gw nightlies seems like a pretty good boon for catching issues early :+1: